blocks/editable/format-toolbar/style.scss

```diff
@@ -41,3 +41,11 @@
 		margin: 0;
 	}
 }
+
+.blocks-url-input {
```

Comment:
If these styles are specific to the `BlocksURLInput` component, they should probably be defined in:

https://github.com/WordPress/gutenberg/blob/master/blocks/url-input/style.scss

Reply:
>If these styles are generic to all `BlocksURLInput` components

Although it seems like we might not want this to be generic to all these components? If it should only be applied within the format toolbar, the styles must be defined to nest as such (since there is no implied scoping for styles defined in this file to make them specific to the format toolbar).

Reply:
I moved this into the toolbar block above.